From 876edcc4cffd26f83eae591e906384dab7f25a51 Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Wed, 26 May 2021 15:30:04 +0900
Subject: [PATCH] cifsd: return zero in always success case
Git-commit: 876edcc4cffd26f83eae591e906384dab7f25a51
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Return zero in always success case.

Reviewed-by: Dan Carpenter <dan.carpenter@oracle.com>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/cifsd/auth.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fs/cifsd/auth.c b/fs/cifsd/auth.c
index adfb3b33f2e5..a89de24aa576 100644
--- a/fs/cifsd/auth.c
+++ b/fs/cifsd/auth.c
@@ -943,7 +943,7 @@ static int generate_smb3signingkey(struct ksmbd_session *sess,
 			SMB2_NTLMV2_SESSKEY_SIZE, sess->sess_key);
 	ksmbd_debug(AUTH, "Signing Key   %*ph\n",
 			SMB3_SIGN_KEY_SIZE, key);
-	return rc;
+	return 0;
 }
 
 int ksmbd_gen_smb30_signingkey(struct ksmbd_session *sess)
@@ -1011,7 +1011,7 @@ static int generate_smb3encryptionkey(struct ksmbd_session *sess,
 		ksmbd_debug(AUTH, "ServerOut Key %*ph\n",
 			SMB3_GCM128_CRYPTKEY_SIZE, sess->smb3decryptionkey);
 	}
-	return rc;
+	return 0;
 }
 
 int ksmbd_gen_smb30_encryptionkey(struct ksmbd_session *sess)
-- 
2.33.0

